File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.swift
import Foundation

func solution(_ arr1:[[Int]], _ arr2:[[Int]]) -> [[Int]] {
    var result: [[Int]] = Array(repeating: Array(repeating: 0, count: arr1[0].count), count: arr1.count)
    for y in 0..<arr1.count {
        for x in 0..<arr1[0].count {
            result[y][x] = arr1[y][x] + arr2[y][x]
        }
    }
    return result
}

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.cpp
#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
    int col = arr1[0].size(), row = arr1.size();

    vector<vector<int>> answer(row, vector<int>(col, 0));

    for (int i = 0; i < row; i++) {
        for (int j = 0; j < col; j++) {
            answer[i][j] = arr1[i][j] + arr2[i][j];
        }
    }

    return answer;
}